test/api/input_file_test.cpp

```diff
@@ -12,7 +12,8 @@ std::string const default_alignment_short_reads_file_path = DATADIR"paired_end_m
 std::string const default_alignment_long_reads_file_path = DATADIR"simulated.minimap2.hg19.coordsorted_cutoff.sam";
 std::filesystem::path const empty_output_path{};
 std::vector<detection_methods> const default_methods{cigar_string, split_read, read_pairs, read_depth};
-constexpr uint64_t sv_default_length = 30;
+constexpr uint64_t default_min_length = 30;
+constexpr uint64_t default_max_overlap = 10;
 
```

Comment:
I believe this should be `uint32_t`? Also why the swtich from `uin64_t` to `uint32_t` everywhere?